class/EditInternshipFormView.php

```diff
@@ -92,10 +92,24 @@ public function buildInternshipForm()
             $this->tpl['DELETE_URL'] = 'index.php?module=intern&action=DeleteInternship&internship_id=' . $this->intern->getId();
         }
 
-        // Determine if we can copy to the next term (i.e. the next term exists)
+        // Get next three terms
         $nextTerm = Term::getNextTerm($this->intern->getTerm());
+        $nextTwoTerm = Term::getNextTerm($nextTerm);
+        $nextThreeTerm = Term::getNextTerm($nextTwoTerm);
+        // Determine if we can copy to the next term (i.e. the next term exists)
         if(Term::termExists($nextTerm)){
-            $this->tpl['NEXT_TERM'] = Term::rawToRead($nextTerm);
+          $this->tpl['NEXT_TERM'] = Term::rawToRead($nextTerm);
+        }
+        //Copy if it's Spring and exist, else if it's Summer 1 and exist. Hide the other term(s)
+        if(Term::termExists($nextThreeTerm) && $this->intern->getTerm()%10 == 1){
```

Comment:
Should use Term::getSemester(..) and the term class constants defined in Term.php (e.g. Term::SPRING).